### ISSUE TITLE
Add grid.column dynamically in CE configuration?

### ISSUE THREAD

**Issue by apiening** [FluidTYPO3/flux#1692]
I've used this CE for accordions in Typo3 7: https://github.com/bootstraptheme-for-typo3/fluidbootstraptheme/blob/development/Resources/Private/Templates/Content/Accordion.html

It does not work in Typo3 9.5.5 (fluid 9.5.5) anymore: If I create multiple panels in the CE configuration and add one CE to one of the columns in the BE, the same CE is displayed in all panels / columns.

I wonder if there is a simple way to fix this by changing the configuration section or does it need to be done entirely different?
Basically I need a dynamic container where I can add additional columns in the CE configuration and display them in the FE.

**Comment by apiening** [comment]
The solution provided works for me. Closing.
